backend/src/review/review.controller.ts

```diff
@@ -158,4 +158,12 @@ export class ReviewController {
   ): Promise<Review[]> {
     return this.reviewService.getLandlordReviews(landlord.landlord);
   }
+
+  // Get landlord name suggestions
+  @Get('/landlord/suggestions')
```

Comment:
There needs to be a Throttle added here for this endpoint. `@Throttle(10, 30)` is likely good.